src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,16 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        StringBuilder lettersAndNums = new StringBuilder();
+        for (int i = 0; i < text.length(); i++) {
+            if (Character.isAlphabetic(text.charAt(i))) {
+                lettersAndNums.append(Character.toLowerCase(text.charAt(i)));
+            } else if (Character.isDigit(text.charAt(i))) {
+                lettersAndNums.append(text.charAt(i));
+            }
```

Comment:
Good job! try to replace that logic with regular expression

Reply:
done